apps/hashdotai/.eslintrc.cjs

```diff
@@ -1,9 +1,4 @@
 /** @type {import("eslint").Linter.Config} */
 module.exports = {
   ...require("@local/eslint-config/generate-workspace-config.cjs")(__dirname),
-  rules: {
-    "simple-import-sort/exports": "error",
-    "simple-import-sort/imports": "error",
-    "unicorn/filename-case": "error",
-  },
```

Comment:
Moved to `libs/javascript/@local/eslint-config/legacy-base-eslintrc-to-refactor.cjs`